### PR TITLE
Add DSTU2 for MedicationPrescription

### PR DESCRIPTION
**Pull request by cerbs**
Adding updated MedicationPrescription doc for DSTU2

### CODE REVIEW

File: lib/resources.rb [hunk 1]
```diff
@@ -1205,6 +1205,53 @@ def get_resource(key)
...
+        "resourceType"=>"Bundle",
+        "id"=>"11025158-de65-4fd2-a7e6-a1f4a2bf897b",
+        "type"=>"searchset",
+        "base"=>
+            "https://fhir.sandboxcernerpowerchart.com/dstu2/open/d075cf8b-3261-481d-97e5-ba6c48d3b41f",
+        "total"=>1,
+        "link"=>
+            [{"relation"=>"self",
+              "url"=>
+                  "https://fhir.sandboxcernerpowerchart.com/dstu2/open/d075cf8b-3261-481d-97e5-ba6c48d3b41f/MedicationPrescription?patient=33840071"}],
```
Comment (dennispatterson) [inline lib/resources.rb]: This patient id has an extra '1'

**Comment by dennispatterson** [comment]
:+1: assuming squash

**Comment by naveenaluru** [comment]
:+1:

**Comment by MaxPhilips** [comment]
+1

**Comment by jfrhong** [comment]
+1

**Pull request merged**